release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -347,11 +347,18 @@ You can now remove or rotate the GCP admin-level credential that the xref:../ope
 
 You can now use the Compliance Operator to perform Center for Internet Security (CIS) Kubernetes Benchmark checks. CIS profiles for {product-title} are based on the CIS Kubernetes checks.
 
+Until the CIS {product-title} Benchmark is published, you can refer to the link:https://pnt.redhat.com/pnt/p-18428165/Red_Hat_OpenS...uide_v1.0.pdf[Red Hat {product-title} Hardening Guide].
```

Comment:
Addressing points 1&2 of https://github.com/openshift/openshift-docs/issues/26801#issuecomment-766500209
@xiaojiey